CodeforAustralia/council-emissions-calculator
# How we calculate (part 2)

## Overview of the Pull Request:

Hey team, 

### This PR includes:
* add text blocks for all sections with content;
* provide links to referenced resources;
* add anchor tags for on-page navigation using the table of content;
* add a link back to the main page;
* run Prettier;

### Out of scope for this PR:
* convert CalculationFormula into text block with cvg`s (next PR);
* calculationFormula is messing up responsiveness atm, will be fixed once converted to text, so please disregard;
* button to go back up to the beginning of the page';
* proof read the text (used the one checked by Sean but just in case :-) 

## How Has This Been Tested?
- Load to your machine;
- Check all links in the table of content;
- Check the link to go back to the results page - now both pages are connected;
- Check the links for all referenced resources:
  - Code for Australia;
  - Civic Makers;
  - GHGProtocol

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

~- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?~
